### ISSUE TITLE
Reformat display name 

### ISSUE THREAD

**Issue by jcahail** [azavea/pfb-network-connectivity#449]
The names of the places currently read: Boulder city, CO, CO; They should be city, state (city) 

e.g.: Boulder, CO (city)

**Comment by CloudNiner** [comment]
Do not add `(city)` to the updated names.

Remove state abbreviation and 'city' text from all neighborhood labels on production.

**Comment by CloudNiner** [comment]
I removed the `' city'` text from all labels on production by doing the following in the console:
```
>>> from django.core.exceptions import ValidationError
>>> for n in Neighborhood.objects.filter(label__contains=' city'):
...     try:
...         n.label = ''.join(n.label.rsplit(' city', 1))
...         n.name = n.name_for_label(n.label)
...         n.save()
...     except ValidationError as e:
...         print n.name + ': ' + n.label
...
```
Use of rsplit so that city names that include ' city', e.g. 'Kansas City city' would not delete the wrong city.

I also manually updated all of the batch analysis shapefiles so that the city names match the updated labels. I'll also be providing internal documentation to our teams which updates which shapefiles should be used to trigger a batch.

**Comment by CloudNiner** [comment]
Closed by #468
